Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1

```diff
@@ -207,26 +207,15 @@
             if ($removeApp)
             {
                 Write-Host (Get-VstsLocString -Key SFSDK_AppAlreadyExistsInfo -ArgumentList @($ApplicationName, $app.ApplicationTypeName, $app.ApplicationTypeVersion))
-
-                try
-                {
-                    $global:operationId = $SF_Operations.RemoveApplication
-                    $app | Remove-ServiceFabricApplication -Force
-                }
-                catch [System.TimeoutException]
-                {
-                    Write-Host (Get-VstsLocString -Key SFSDK_PerformingForceRemoveOnTimeout -ArgumentList $ApplicationName)
-                    $app | Remove-ServiceFabricApplication -Force -ForceRemove
-                }
+                Remove-ServiceFabricApplicationAction -ApplicationName $($app.ApplicationName)
```

Comment:
Where have you implemented Remove-ServiceFabricApplicationAction ?
Please include the change in this PR

Reply:
That function was reverted in merge. I have brought the changes back.